codespell_lib/data/dictionary.txt

```diff
@@ -11412,9 +11419,11 @@ examinated->examined
 examing->examining
 examinining->examining
 examles->examples
+examnples->examples
 exampel->example
 exampeles->examples
 exampels->examples
+examplee->example
```

Comment:
This should probably suggest the plural too (in case the second e should be an s). Can you also add another line with the plural version of this typo too.

Reply:
adding both.